drivers/spi/spi_pl022.c

```diff
@@ -482,8 +482,9 @@ static void spi_pl022_xfer(const struct device *dev)
 		SSP_READ_REG(SSP_DR(cfg->reg));
 
 	while (data->rx_count < chunk_len || data->tx_count < chunk_len) {
+		size_t tx_cnt = 0;
```

Comment:
It looks like the `tx_cnt` also counts the number of bytes received.
Can you reconsider this name?
(I also hesitated and chose `txrx` above. If you have a better name, please.)

Reply:
Named it `fifo_cnt`. In case if `txrx` I don't have a better name, but you could leave the variable away and do
```c
if (txbuf) {
    SSP_WRITE_REG(SSP_DR(cfg->reg), ((uint8_t *)txbuf)[data->tx_count]);
} else {
    SSP_WRITE_REG(SSP_DR(cfg->reg), 0);
}
```